windows/check_quiesce_snapshot/check_quiesce_snapshot.yml

```diff
@@ -44,6 +44,13 @@
             - include_tasks: ../../common/vm_take_screenshot.yml
               vars:
                 vm_take_screenshot_local_path: "{{ current_test_log_folder }}"
+            
+            # Taking quiesce snapshot on Windows Server takes longer sometime
+            - name: "Pause 10 seconds after taking quiesce snapshot"
+              pause:
+                seconds: 10
+              when: guest_os_product_type | lower == "server"
+ 
```

Comment:
How about adding a wait_for task like waiting for vss log appear?

Reply:
Nothing in vmware.log can be used to indicate this clearly, then changed to use wait until specified file generated and exists here. Thanks.